app/controllers/users_controller.rb

```diff
@@ -66,4 +67,18 @@ def statistics
               owner(current_user.id).
               latest
   end
+
+  private
+
+  def user_slide_with_paginate(user_id)
+    if params[:sort_by] == 'popularity'
```

Comment:
__[rubocop]__
* Style/ConditionalAssignment: Use the return of the conditional for variable assignment and comparison.